swift/IceDiscovery/greeter/Sources/Client/main.swift

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) ZeroC, Inc.
+
+import Foundation
+import Ice
+import IceDiscovery
+
+// Configure the communicator to load the IceDiscovery plugin during initialization. This plugin installs a default
+// locator on the communicator.
+var args = CommandLine.arguments
+let properties = try Ice.createProperties(&args)
+var initData = Ice.InitializationData()
+initData.properties = properties
+initData.pluginFactories = [IceDiscovery.discoveryPluginFactory()]
```

Comment:
There is no pluginFactories  field in Ice for Swift.

You should instead set the property Ice.Plugin.IceDiscovery to 1, like in the matlab/IceDiscovery/greeter demo:
```swift
properties.setProperty(key: "Ice.Plugin.IceDiscovery, value: "1")
```